addon/components/frost-link.js

```diff
@@ -1,70 +1,59 @@
 import Ember from 'ember'
 const {
+  deprecate,
+  get,
   LinkComponent,
   Logger,
-  deprecate,
-  get
+  set
 } = Ember
 import computed, {readOnly} from 'ember-computed-decorators'
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
 import layout from '../templates/components/frost-link'
 
+/**
+ * List of valid values to pass into `design` propery
+ * @type {Array} valid `design` values
+ */
 const validDesigns = [
   'info-bar',
   'in-line',
   'inline'
 ]
```

Comment:
Are we dropping `design`?  I believe we had discussed that with respect to the button